frontend/src/scenes/funnels/Funnel.scss

```diff
@@ -1,30 +1,5 @@
 @import '~/vars';
 
-.funnels-empty-state {
```

Comment:
We remove all these local override styles because we now standardize at the insight level